src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/misc/antibot/modes/CustomAntiBotMode.kt

```diff
@@ -111,11 +113,15 @@ object CustomAntiBotMode : Choice("Custom"), ModuleAntiBot.IAntiBotMode {
             ArmorConfigurable("Leggings", EquipmentType.LEGGINGS),
             ArmorConfigurable("Chestplate", EquipmentType.CHESTPLATE),
             ArmorConfigurable("Helmet", EquipmentType.HELMET),
-        ).onEach(::tree)
+        )
+
+        init {
+            values.reversedArray().onEach(::tree)
```

Comment:
It is better to deploy an array once and add it than to deploy an array of armor items each time (123 line `armorItems.reversed()`)